pages/common/vim.md

```diff
@@ -18,7 +20,11 @@
 
 - Quit without saving:
 
-`:quit!<Enter>`
+`:quit!<Enter>` or `:q!<Enter>`
```

Comment:
We prefer the long format to be easier for beginners.

Reply:
That's a fair point.